hypha/apply/users/templates/two_factor/core/backup_tokens.html

```diff
@@ -1,26 +1,37 @@
 {% extends "two_factor/_base_focus.html" %}
-{% load i18n users_tags %}
+{% load static i18n users_tags %}
 
 {% block content %}
-  <p><a href="{% url 'users:account'%}"
-       class="btn btn-link">{% trans "Back to account" %}</a></p>
+  <script src="{% static 'js/clipboard.js' %}"></script>
+  <p class="actions actions-header"><a href="{% url 'users:account' %}" class="link link--bold">{% trans "Back to Account" %}</a></p>
   <h1>{% block title %}{% trans "Backup Codes" %}{% endblock %}</h1>
-  <p>{% blocktrans trimmed %}These codes should be kept in a safe,
-      private place for when you need them. When they are used up,
-      you can generate a new set of backup codes.{% endblocktrans %}</p>
+  <p>{% blocktrans trimmed %}Each of the code can be used only once. When they are used up, you can generate a new set of backup codes.{% endblocktrans %}</p>
 
   {% if device.token_set.count %}
-    <ul>
-      {% for token in device.token_set.all %}
-        <li>{{ token.token }}</li>
-      {% endfor %}
-    </ul>
-    <p>{% blocktrans %}You should now download, or print these codes,
-        and keep them somewhere safe.{% endblocktrans %}</p>
-    <form method="post">{% csrf_token %}{{ form }}
+<textarea readonly cols="8" rows="{{ device.token_set.count }}" id="list-backup-tokens">{% for token in device.token_set.all %}{{ token.token }}{% if not forloop.last %}&#013;&#010;{% endif %}{% endfor %}</textarea>
+    <p class="hide-print">{% blocktrans %}You should now print these codes or copy them to your
+       clipboard and store them in your password manager.{% endblocktrans %}</p>
+
+    <form method="post" class="actions actions-footer">{% csrf_token %}{{ form }}
+    <p>
+        <a href="#" class="btn btn-primary btn-print">{% trans "Print" %}</button>
+        <a href="#"
+            class="btn btn-primary btn-copy-to-clipboard"
+            data-clipboard-target="#list-backup-tokens"
+            data-clipboard-action="copy"
+            data-tippy-content="{% trans "Copied!" %}"
+        >{% trans "Copy to Clipboard" %}</a>
         <button class="btn btn-primary" type="submit">{% trans "Regenerate Codes" %}</button>
-        <a class="btn btn-link link--left-space" href="data:text/plain;charset=UTF-8,{% tokens_text device.token_set.all %}" download="backup_codes.txt">
-        {% trans "Save Codes" %}</a>
+    </p>
+
+
+    <p class="hide-print">{% blocktrans %}Once done, acknowledge you have stored the codes securely and then click "Finish".{% endblocktrans %}</p>
+    <div class="form">
+        <ul class="errorlist hidden error-action-agree"><li>Please confirm you have stored the codes securely below.</li></ul>
+        <input type="checkbox" id="action_agree" name="action_agree" value="action_agree">
```

Comment:
This checkbox is not being hidden, it shows above the form.
![Skärmavbild 2022-11-14 kl  12 45 52](https://user-images.githubusercontent.com/191392/201652306-f1b5bdc2-b1af-4430-9b70-ca6e8cab35ee.png)

Reply:
okay will take a look.

Reply:
Fixed.

![image](https://user-images.githubusercontent.com/236356/202433246-7501aaa6-bf4f-47cb-94e9-2728f9b4dda6.png)